entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -l
+
+SOOS_APP_VERSION=${GITHUB_ACTION_REF}
+
+SOOS_INTEGRATION_NAME="GitHub"
+SOOS_INTEGRATION_TYPE="Plugin"
+
+SOOS_BRANCH_NAME=${SOOS_BRANCH_NAME:-${GITHUB_REF}}
+if [[ -n $SOOS_SAST_PATH ]]; then
+    SOOS_SOURCE_CODE_PATH=${GITHUB_WORKSPACE}/${SOOS_SAST_PATH}
+else
+    SOOS_SOURCE_CODE_PATH=${GITHUB_WORKSPACE}
+fi
+
+PARAMS=(
+    "--apiKey ${SOOS_API_KEY}"
```

Comment:
these don't line up with https://github.com/soos-io/soos-sca-github-action/pull/17/files#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2 - needs double quote too

Reply:
![image](https://github.com/soos-io/soos-sast-github-action/assets/88039131/04d6dc65-41c1-4c5d-b81d-4c9b7377c4b6)
we should get them to be the same - if we don't need those around api key etc, we can remove them from sca